src/lib.rs

```diff
@@ -543,6 +551,14 @@ impl<H, N, S, Id> HistoricalVotes<H, N, S, Id> {
 	}
 }
 
+pub trait AccountableSafety {
```

Comment:
This is unused and will probably need to be changed once it's actually used (no particular reason it's just the way things usually go). Maybe return `impl Iterator` instead? I'm fine with adding it anyway.

Reply:
It seems we can't return `impl Iterator` (unless I box it?) because is a trait method.  I changed it to return a `Self::Messages` for now...